lbrynet/daemon/ExchangeRateManager.py

```diff
@@ -43,11 +42,16 @@ def __init__(self, market, name, url, params, fee):
         self.fee = fee
         self.rate = None
         self._updater = LoopingCall(self._update_price)
+        self._online = True
 
     @property
     def rate_is_initialized(self):
         return self.rate is not None
 
+    @property
+    def is_online(self):
```

Comment:
why not make this a function? whats the benefit of hiding the fact that a function is being called? i see this `@property` stuff everywhere in python, but i dont understand it at all

Reply:
i don't see a good reason for using @property in either is_online() or rate_is_initialized() so I will remove them